### PR TITLE
Add Spark 3.4

### PR DESCRIPTION
**Pull request by chethanuk**

### CODE REVIEW

**Review by eddumelendez — APPROVED** [review]

File: src/main/scala/io/sdkman/changelogs/SparkMigrations.scala [hunk 1]
```diff
@@ -33,4 +33,19 @@ class SparkMigrations {
       .insert()
       .asCandidateDefault()
   }
+  
+  @ChangeSet(
+    order = "025",
+    id = "025-add_spark_3.4.2",
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/SparkMigrations.scala:39]:
```suggestion
    id = "025-add_spark_3.4.0",
```

File: src/main/scala/io/sdkman/changelogs/SparkMigrations.scala [hunk 2]
```diff
@@ -33,4 +33,19 @@ class SparkMigrations {
       .insert()
       .asCandidateDefault()
   }
+  
+  @ChangeSet(
+    order = "025",
+    id = "025-add_spark_3.4.0",
+    author = "sekikn"
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/SparkMigrations.scala:40]:
```suggestion
    author = "chethanuk"
```

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
Thanks @chethanuk !

**Comment by chethanuk** [comment]
Hey @eddumelendez When will 3.4 be avilable?
```
Available Spark Versions
================================================================================
     3.3.2
```

**Comment by marc0der** [comment]
Should have been immediately unless something wasn't right with the PR.

**Comment by eddumelendez** [comment]
looks like it failed because of formatting but it was not caught in the PR 🤔  https://github.com/sdkman/sdkman-db-migrations/actions/runs/5171523351/jobs/9315106409

**Comment by marc0der** [comment]
Thought I'd fixed that in the workflow 🤔

**Comment by marc0der** [comment]
If you can run scalafmt and repush your pr branch we can merge it asap.

**Pull request merged**